ODLA/platforms/dnnl/odla_dnnl.cc

```diff
@@ -328,6 +329,14 @@ odla_status odla_ExecuteComputation(odla_computation comp, odla_context context,
     op.execute(*context->stream);
   }
   context->stream->wait();
+  // copy to outputs
+  auto outputs_v = context->comp->outputs_v;
+  for (auto& output_pair : outputs_v) {
+    auto& src_val = output_pair.second.first;
+    auto& dst_ptr = output_pair.second.second;
+    memcpy(dst_ptr, src_val->mem.get_data_handle(),
+           src_val->mem.get_desc().get_size() * (src_val->elem_size / 4));
```

Comment:
why do we need to "/ 4"?  elem_size is number of bytes per element

Reply:
it's for handling int64 outputs.

Reply:
can we just use "src_val->mem.get_desc().get_size()" ?

Reply:
> can we just use "src_val->mem.get_desc().get_size()" ?

That may fail some test cases, when input value is ODLA_INT64, it actually create a dnnl::memory::data_type::s32 memory object, so if we want to get this data as output, get_size() doesn't give the correct size.

Reply:
so should it be something like "GetTotalElements(src_val) * src_val->elem_size " ?

Reply:
yes

Reply:
ok. let's use  "GetTotalElements(src_val) * src_val->elem_size " to avoid confusion.

Otherwise, looks good to me.